benchmarks/cuda/benchmark_cuda.cpp

```diff
@@ -6,13 +6,17 @@
  */
 
 // VecMem include(s).
+#include <vecmem/memory/binary_page_memory_resource.hpp>
 #include <vecmem/memory/cuda/device_memory_resource.hpp>
 #include <vecmem/memory/cuda/host_memory_resource.hpp>
 #include <vecmem/memory/cuda/managed_memory_resource.hpp>
 
 // Google benchmark include(s).
 #include <benchmark/benchmark.h>
 
+#include <iostream>
```

Comment:
I guess this was left here by mistake. :stuck_out_tongue:

Reply:
Indeed it was, fixed.